policies/code-of-conduct.md

```diff
@@ -66,32 +55,41 @@ If you're interested in an introduction to diversity, inclusion, and unconscious
+
 #### Leadership Expectations
 
-There are further expectations for members of leadership teams including the CFC.
+Leaders should act with respect, engage directly, and be willing to resolve conflicts. See [Martha's Rules][] and the [Decision Making][] section of our bylaws for examples of how to help constructively resolve disagreement.
 
-When decisions are made within the established guidelines and policies of CF projects, *those in leadership roles have a responsibility to uphold and respect the decision even if they disagree with it*. Should the leader be unwilling or unable to do so, then they should resign their leadership position.
+CF Leadership is also responsible for ensuring that decisions, once made, are supported and revisited only when necessary to uphold the integrity of the collaborative process.
 
-This does not mean that decisions are set in stone. CF projects and the CFC are always able to revisit decisions at a later time.
+[Martha's Rules]: https://digitalcommons.unl.edu/cgi/viewcontent.cgi?article=1825&context=sociologyfacpub
+[Decision Making]: ../bylaws/5-decision-making.md
 
 ### Report an issue
 
-- Each CF project is free to implement enforcement of the CoC using processes appropriate for the size and scope of the project. CoC enforcement processes and contact methods must be defined in the projects' `CODE_OF_CONDUCT.md`.
-- For CF repositories that do not specify their own the `CODE_OF_CONDUCT.md`, send your report to the [`report` mailing list][CONTACTS.yaml]. The CFC will receive these reports.
+Disagreements and conflicts are inherent to any dynamic community. We encourage direct dialogue as the first step toward resolution, recognizing that constructive disagreement can lead to innovation and growth.
 
-Confidentiality is of utmost importance. We also understand that the person reporting the issue (**reporter**) may not be the individual directly affected or harmed (**target**). In such cases, communication will primarily be with the reporter unless the target explicitly gives permission. Personal details of the reporter (and the target if they differ) will not be disclosed without permission.
+For issues that escalate beyond direct resolution or involve behavior contrary to our Code of Conduct, the CF provides clear avenues for reporting:
 
-- The CF *prohibits retaliation* against individuals who report concerns in good faith.
+- **Project-Specific Reporting**: Each CF project may implement its own CoC enforcement processes suitable to its size and scope. These processes and contact methods should be detailed in the project's `CODE_OF_CONDUCT.md`.
+- **Foundation-Wide Reporting**: For concerns within CF spaces not covered by a specific project's `CODE_OF_CONDUCT.md`, or for issues that require escalation beyond the project level, reports should be directed to the [report mailing list][CONTACTS.yaml]. The CFC will oversee these reports.
```

Comment:
Should this be clearer to indicate projects without a specific CoC are covered by the CF one? "CF spaces", to me, left room for uncertainty.

One possible alternative is "For concerns within the CF, projects not covered by their own CoC, or for issues that require escalation beyond the project level"?

Reply:
sure. suggest? I was smashing a lot of text..

Reply:
Took a stab at it by just removing the extra words.. WDYT?

Reply:
I suggested:
```
For concerns within the CF, projects not covered by their own CoC, or for issues that require escalation beyond the project level, reports should be directed to the [report mailing list][CONTACTS.yaml]. The CFC will oversee these reports.
```

Reply:
oh! yes. you didn't use the +- thing so I could just accept it. Sorry. I read too fast. ;)

Reply:
What you have now is good!